src/.nuget/dir.props

```diff
@@ -112,9 +112,6 @@
 
   <ItemGroup Condition="$(SupportedPackageOSGroups.Contains(';Linux;'))">
     <OfficialBuildRID Include="alpine.3.4.3-x64" />
```

Comment:
Why not remove the other non-portable RID's as well? Similar to what I'm doing in corefx?